_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,424 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $helper-text;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(x-small);
+        }
+
+        .helper-text { // Generally used for radios and checkboxes
+            margin-top: 0;
```

Comment:
Looks like this actually should have a margin top, though the default is too much. Maybe about .3rem?